src/argilla/server/schemas/v1/datasets.py

```diff
@@ -384,19 +384,31 @@ class VectorsSettings(BaseModel):
     items: List[VectorSettings]
 
 
-class VectorSettingsCreate(BaseModel):
-    name: str = PydanticField(
-        ...,
+VectorSettingsName = Annotated[
```

Comment:
If you are not using `VectorSettingsName` in any other place I think you should define it directly inside `VectorSettingsCreate` model.

Something like:

```python
class VectorSettingsCreate(BaseModel):
    title: Annotated[
        constr(
            regex=VECTOR_SETTINGS_CREATE_NAME_REGEX,
            min_length=VECTOR_SETTINGS_CREATE_NAME_MIN_LENGTH,
            max_length=VECTOR_SETTINGS_CREATE_NAME_MAX_LENGTH,
    )
```